extensions/apicurio-registry-avro/deployment/src/main/java/io/quarkus/apicurio/registry/avro/ApicurioRegistryAvroProcessor.java

```diff
@@ -55,6 +55,12 @@ public void apicurioRegistryAvro(BuildProducer<ReflectiveClassBuildItem> reflect
                 "io.apicurio.rest.client.auth.Auth",
                 "io.apicurio.rest.client.auth.BasicAuth",
                 "io.apicurio.rest.client.auth.OidcAuth"));
+
+        // Confluent API compatibility support -- TODO
+
+        reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, false,
```

Comment:
wondering if we need to add a flag to enable/disable this.

Reply:
Typically, guarded with `isCcompat`

Reply:
Is this actually even needed?
Since the IT (native) test worked before I added this ...

I just added this not to forget, and was hoping IT (native) test would expose all the missing stuff ...

Reply:
Actually, that's a good question. What happens if we include in the native image classes that are not available in the classpath. Would that fail? I don't remember... (that's what I'm trying to prevent).

Reply:
Looking at KafkaProcess, all these registrations are guarded by try/catch on the CNFE, with a previous try to load a class from the lib it's trying to support.
So maybe this try/catch is missing / needed here as well?